binary128/binary128_test.go

```diff
@@ -1,10 +1,9 @@
 package binary128
 
 import (
-	"testing"
 	"math"
-	"math/big"
 	"github.com/stretchr/testify/assert"
+    "testing"
```

Comment:
So sort imports, try running `goimports -w binary128_test.go`

First you may need to install `goimports` by running `go get -u golang.org/x/tools/cmd/goimports`

Reply:
It looks like all files could benefit from `goimports`

Lint issues from https://travis-ci.org/mewmew/floats/builds/399079263?utm_source=github_status&utm_medium=notification

```
binary128/binary128.go:30: File is not goimports-ed (goimports)
	sign := intRep & 0x80000000 != 0
binary128/binary128_test.go:8: File is not goimports-ed (goimports)
binary128/binary128_test.go:4: File is not gofmt-ed with -s (gofmt)
	"math"
```